sheet_foot.rst

```diff
@@ -2,6 +2,7 @@
 Additional Resources
 ********************
 
-RST also has many more features.  Check out the official `RST User Documentation`_.
+Both MD and RST have many more features, please follow the README_ to add more examples.  Also check out the official `RST User Documentation`_.
```

Comment:
```suggestion
Both MD and RST have many more features.  Also check out the official `RST User Documentation`_.
```
It's understood that you can contribute to a repo by reading the README.  These sentences are aimed at people needing help with RST, so we should keep it direct.

Reply:
I wanted to encourage adding more examples and the compilation process for this is different enough I thought it was worth calling out, though it is included in a comment at the top of the combined file.